docs/source/researcher_resources.rst

```diff
@@ -0,0 +1,116 @@
+Documentation Index for Researchers
+===================================
+
+.. toctree::
+   :maxdepth: 3
+   :caption: Contents:
+
+
+The Vivarium Framework
+++++++++++++++++++++++
+
+The Vivarium microsimulation framework is separated into numerous source-controlled repositories to keep things focused
+and organized. A list of repositories relevant to researchers can be found below, including short descriptions and links
+to the repositories and their documentation. Each repository holds installation information in its README.
+
+
+Vivarium
+--------
+
+Vivarium is the core microsimulation framework and command line tools for running individual simulations. It contains
+documentation about how Vivarium itself works as well as useful information about how a model is specified and tutorials
+for running simulations and exploring them interactively, among other things.
+
+- `Repository <https://github.com/ihmeuw/vivarium>`_
+- `Documentation <https://vivarium.readthedocs.io/projects/vivarium/en/latest/>`_
+
+
+Vivarium Public Health
+----------------------
+
+Vivarium Public Health is a set of tools for Vivarium simulations that enable public health applications like disease
+and intervention modeling. This is where we define things like SI- or SIS epidemiological models, or fertility models
+for population growth. It also contains a useful tutorial on interacting with data artifacts.
+
+- `Repository <https://github.com/ihmeuw/vivarium_public_health>`_
+- `Documentation <https://vivarium.readthedocs.io/projects/vivarium-public-health/en/latest/>`_
+
+
+Vivarium Inputs
+---------------
+
+Vivarium Inputs contains the tools needed to create data artifacts from GBD data. The data artifact is a useful notion,
+you can think of it as all of the data needed to run a simulation, packaged into one file. This package contains useful
+functions for pulling raw GBD data as well as generating data as it is used in a simulation, which are vitally important
+for vetting.
```

Comment:
I like the specific mention of the data artifact tutorial in the vph section - can we specifically call out the vi pulling data tutorial here as well?